primitives/runtime/src/generic/digest.rs

```diff
@@ -82,10 +82,14 @@ impl<Hash> Digest<Hash> {
 #[cfg_attr(feature = "std", derive(parity_util_mem::MallocSizeOf))]
 pub enum DigestItem<Hash> {
 	/// System digest item that contains the root of changes trie at given
-	/// block. It is created for every block iff runtime supports changes
+	/// block. It is created for every block if runtime supports changes
```

Comment:
`iff` is not a typo and means "if and only if".

Reply:
:sweat_smile: